3회차/여다영/5_신용카드만들기2.py

```diff
@@ -1,3 +1,20 @@
 import sys
 
 sys.stdin = open("_신용카드만들기2.txt")
+
+T = int(input())
+
+for i in range(1, T + 1):
+    credit_nums = input()
+    if credit_nums[0] == '3' or credit_nums[0] == '4' or credit_nums[0] == '5' or credit_nums[0] == '6' or credit_nums[0] == '9':
```

Comment:
`in` 구문을 활용해서 코드를 짜보면 어떨까요?
```suggestion
    if credit_nums[0] in '34569':
```